### PR TITLE
Finalisation Css Pod

### PR DESCRIPTION
**Pull request by CecileLehoux**

### CODE REVIEW

File: src/Astro.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable no-unused-vars */
+import axios from 'axios';
+import { useState } from 'react';
+import AstronautDetails from './components/AstronautDetails';
+
+function Astronautes() {
```
Comment (Froyj) [inline src/Astro.jsx:7]: `function Astronauts() {`

File: src/components/Home.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Link } from 'react-router-dom';
 import presentationiss from '../ressources/assets-home/presentation-iss.png';
 import isstracker from '../ressources/assets-home/iss-tracker.png';
 import celestobjects from '../ressources/assets-home/celest-objects.png';
-import pod from '../ressources/assets-home/pod.png';
+import podIcone from '../ressources/assets-home/podIcone.png';
```
Comment (Froyj) [inline src/components/Home.jsx:6]: podIcon

File: src/components/ApiTeamISS.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import TeamISSCard from './TeamIssCard';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 980px;
+  justify-content: space-around;
+  margin-top: 2em;
+  margin-bottom: 4em;
+`;
```
Comment (Froyj) [inline src/components/ApiTeamISS.jsx:14]:
```suggestion
const Container = styled.div`
  display: flex;
  flex-direction: row;
  width: ${((props) => props.width || '100%')};
  justify-content: ${(props) => props.justifyContent};
  margin-top: 2em;
  margin-bottom: 4em;
`;
```

**Review by Froyj — APPROVED** [review]

**Review by YoMhz — APPROVED** [review]
Ok super nikel !

**Pull request merged**